ner/src/main/java/edu/illinois/cs/cogcomp/ner/ExpressiveFeatures/GazetteerTree.java

```diff
@@ -35,7 +36,16 @@
     private StringSplitterInterface splitter = new StringSplitterInterface() {
         @Override
         public String[] split(String line) {
-            return line.split("[\\s]+");
+
+            // character tokenization for Chinese
+            if(ParametersForLbjCode.currentParameters.language.equals("zh")) {
```

Comment:
ok. I have an enum of languages in [cross-lingual wikifier](https://github.com/cttsai/illinois-cross-lingual-wikifier/blob/master/src/main/java/edu/illinois/cs/cogcomp/xlwikifier/datastructures/Language.java). I can do the same here.